### PR TITLE
Adding logo to event map

### LINKED ISSUE #1414: Add TreesCount logo and change text in event PDF
Currently, it says TreesCount! 2015 Event Map. Add in the logo before Event Map.

For you @jfrankl  - I think. @RickMohr is the person to talk to about PDFs.

### PR DESCRIPTION
**Pull request by jfrankl**
That's not a bad looking event map! Adds new content from Deb and adds logo. Fixes #1414.

Thanks for pairing with me on this Rick.

image

### CODE REVIEW

File: src/nyc_trees/apps/event/templates/event/printable_event_map.html [hunk 1]
```diff
@@ -3,12 +3,25 @@
...
+                <p class="map-pdf-subtext">Remember to bring the Census Gear Pack that you received at your training event!</p>
+            {% endif %}
+            </p>
+            <p class="map-pdf-subtext" style="overflow: hidden; height: 1.25in">
+                {{ event.location_description }}
+            </p>
+
         </div>
-        <div id="map" class="map-pdf"
+        <div id="map" class="map-pdf" style="top: 3in"
```
Comment (RickMohr) [inline src/nyc_trees/apps/event/templates/event/printable_event_map.html]: I think this should be `3.5in` to match line 6.

**Comment by RickMohr** [comment]
Otherwise +1

**Comment by RickMohr** [comment]
Will the changes for the Reservations map be in a different PR?

**Comment by jfrankl** [comment]
They're in this PR.

**Comment by jfrankl** [comment]
@RickMohr Did you not test the reservations print map? If not, could you give it a whirl and let me know if it works right?

**Comment by RickMohr** [comment]
@jfrankl, reservations map looks good. Sorry I missed it the first time.

**Comment by jfrankl** [comment]
I should have been more clear in the PR.

**Comment by jfrankl** [comment]
No worries!

On Mon, Apr 27, 2015 at 11:18 AM, Rick Mohr <notifications@github.com>
wrote:

> @jfrankl <https://github.com/jfrankl>, reservations map looks good. Sorry
> I missed it the first time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/nyc-trees/pull/1447#issuecomment-96699409>.
>



-- 
Jeff Frankl, UI/UX Designer

Azavea  |  340 N 12th St, Suite 402, Philadelphia, PA
Web azavea.com  |  Blog azavea.com/blogs  |  Twitter @azavea
<http://twitter.com>

**Pull request merged**